ui/src/message_popup/playtest/smith_experience_page.jsx

```diff
@@ -0,0 +1,221 @@
+/* 
+Experience page for the Mr. Smith Microaggression simulation
+Code adapted from the Turner_scenarios.js file
+MIT Teaching Systems Lab
+ */
+
+import React from 'react';
+import uuid from 'uuid';
+
+import * as Api from '../../helpers/api.js';
+import LinearSession from '../linear_session/linear_session.jsx';
+import SessionFrame from '../linear_session/session_frame.jsx';
+import IntroWithEmail from '../linear_session/intro_with_email.jsx';
+import {SmithScenarios} from './smithTurner_scenarios.jsx';
+import type {QuestionT} from './smithTurner_scenarios.jsx';
+import type {ResponseT} from './smithTurner_scenarios.jsx';
+import VideoSummary from './video_summary.jsx';
+import InstantResponseScenario from '../renderers/instant_response_scenario.jsx';
+import MinimalTextResponse from '../renderers/minimal_text_response.jsx';
+import MixedQuestion from '../renderers/mixed_question.jsx';
+import ChoiceForBehaviorResponse from '../renderers/choice_for_behavior_response.jsx';
+import * as Routes from '../../routes.js';
+
+type NextQuestionT = {
+  id:number,
+  question:QuestionT,
+  responses:[ResponseT]
+};
+
+// The top-level page, manages logistics around email, questions,
+// and the display of instructions, questions, and summary.
+export default React.createClass({
```

Comment:
This looks good to work from for now, let's go with that.  It'd be good to look at this in more detail, and see if we can build on `BubbleScenario` instead of the video one here, but we can do that after the playtest.